8Knot/queries/issue_assignee_query.py

```diff
@@ -45,9 +45,11 @@ def issue_assignee_query(self, repos):
                         explorer_issue_assignments ia
                     WHERE
                         ia.id in %s
-                        and ia.created < now()
-                        and ia.closed < now()
-                        and ia.assign_date < now()
+                        and (ia.created < now() or ia.created IS NULL)
```

Comment:
Created should(cant?) never be null as if it was never created it doesnt exist

Reply:
tru